packages/phone-number-privacy/combiner/src/server.ts

```diff
@@ -37,6 +37,14 @@ export function startCombiner(config: CombinerConfig, kit?: ContractKit) {
   // (https://github.com/celo-org/celo-monorepo/issues/9809)
   app.use(express.json({ limit: '0.2mb' }), loggerMiddleware(config.serviceName))
 
+  // enable cross origin resource sharing from any domain so odis can be interacted with from web apps
+  app.use((_, res, next) => {
+    res.header('Access-Control-Allow-Origin', '*')
+    res.header('Access-Control-Allow-Methods', 'DELETE, PUT, GET, POST, OPTIONS')
```

Comment:
I think ODIS only uses `GET` and `POST` (and in-fact may actually only use `POST`) and so there is no need to add `DELETE` and `PUT`, unless you have seen it used and I missed it.
```suggestion
    res.header('Access-Control-Allow-Methods', 'GET, POST, OPTIONS')
```